lib/thrift.ex

```diff
@@ -21,4 +21,8 @@ defmodule Thrift do
 
   @typedoc "Thrift message types"
   @type message_type :: :call | :reply | :exception | :oneway
+
+  def primitive_names do
```

Comment:
Could you add a short `@doc` for this one given that it's in the top-level module?